tests/lib/rules/require-optimization.js

```diff
@@ -47,7 +47,7 @@ ruleTester.run('react-require-optimization', rule, {
         import React, {Component} from "react";
         @reactMixin.decorate(PureRenderMixin)
         class YourComponent extends Component {
-          componetnDidMount () {}
+          componentDidMount () {}
```

Comment:
I’m surprised this didn’t affect the test result

Reply:
This hook isn't essential to the test, the rule doesn't look at it.